Updated ipnetwork to 0.21

fenollp opened the pull request:
This PR updates the ipnetwork dependency to also allow ipnetwork 0.21.

weiznich reviewed (approved):
Thanks for submitting :+1:

(Rebased to pull in the CI fixes)

fenollp commented:
Thanks for merging :)
Can this be part of the next v2 release and when would this be?

I see many changes since the last release https://github.com/diesel-rs/diesel/compare/v2.2.6...master

weiznich commented:
This will be part of the next 2.2.x patch release. We generally do not give any ETA's other than "when it's done" when releases happen as this is still a free time project.
